specificatie/openapi.yaml

```diff
@@ -648,6 +810,174 @@ components:
+
+    AdresseerbaarObjectStatusZoekObject:
+      type: object
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:    
+              type: array
+              minItems: 1
+              items:
+            $ref: '#/components/schemas/AdresseerbaarObjectStatus_enum'
+
+    GebruiksdoelZoekObject:
+      type: object
+      properties:
+        operator:
+          $ref: '#/components/schemas/ArrayOperator'
+        waarden:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Gebruiksdoel_enum'
+
+    AdresseerbaarObjectTypeZoekObject:
+      type: object
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:    
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/TypeAdresseerbaarObject_enum'
+    
+    PandenZoekObject:
+      allOf:
+        - $ref: '#/components/schemas/GeoJsonZoekOpdracht'
+        - type: object
+          description: Een object waarin de benodigde zoek criteria worden opgegeven om naar panden te zoeken.
+          properties:
+            status:
+              description: Een lijst met enumeratie waarden waar gezochte panden wel of niet aan moeten voldoen.
+              $ref: '#/components/schemas/PandStatusZoekObject'
+            geconstateerd:
+              description: Een indicatie of er naar geconstaeerde resources gezocht moet worden.
+              type: boolean
+            bouwjaar:
+              description: Een minimale en/of maximale waarde van het bouwjaar van paden waarnaar gezocht moet worden. 
+              $ref: '#/components/schemas/NumberOperator'
+
+    PandStatusZoekObject:
+      type: object
+      required:
+        - waarden
+      properties:
+        operator:
+          $ref: '#/components/schemas/EnumOperator'
+        waarden:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/StatusPand_enum'
+
+    NumberOperator:
+      description: >-
+        "Een generieke aanduiding voor de min en max waarde van een attribuut van een object.
+        Als alleen een min waarde wordt opgegeven, dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan de min waarde.
+        Als alleen een max waarde wordt opgegeven, dan wordt er gezocht naar waarden die kleiner dan of gelijk zijn aan de max waarde.
+        Als zowel min en max waarde worden opgegeven waarbij min < max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die tussen min en max liggen.
+        Als zowel min en max waarde worden opgegeven waarbij min > max dan wordt er gezocht naar waarden die groter dan of gelijk zijn aan min en kleiner dan of gelijk aan max, oftewel waarden die niet tussen min en max liggen." 
```

Comment:
Zie boven voor een aangepaste description voorstel, en mijn vraag of we nr 4 wel moeten willen.

Reply:
Optie 4 is niet fout, het betekent alleen dat je tot een waarde EN vanaf een waarde kunt zoeken, waarbij de waarde tot waar je zoek kleiner is dan de waarde vanaf waar je zoekt. In dat geval zoek je juist niet tussen de vanaf en tot waarde, maar alles daarbuiten.
Je kunt ook zoeken vanaf een waarde EN tot een waarde, waarbij de waarde vanaf waar je zoekt kleiner is dan de waarde tot waar je zoekt. In dat geval zoek je tussen vanaf en tot waarden.

Reply:
Ik begrijp hoe het werkt, maar twijfel of hier behoefte aan is. Kun je als consumer goed overzien wat je dan zoekt? En hoe groot/breed de zoekvraag is? Heb je een voorbeeld van wanneer je dit zou willen?

Reply:
Dit is nu verwijderd voor alle types, maar voor integers (bv. bouwjaar en oppervlakte) blijft m.i. dan iets onbruikbaars over, mijn voorstel zou zijn om tenminste voor deze gegevens wel filter operators te specificeren.

Reply:
IntegerFilter toegevoegd met min, max en not operators.